### PR TITLE
Phase 1 flashcards

### PR DESCRIPTION
**Pull request by Omicron3141**
It looks ugly but it works

### CODE REVIEW

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -1,17 +1,50 @@
+<script>
+function drawFlashcard() {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    console.log(this.status);
+    if (this.readyState == 4 && this.status == 200) {
```
Comment (kcwoolverton) [inline app/views/courses/show.html.erb:6]: Out of curiosity, what do 4 and 200 do/mean in this context?

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -1,17 +1,50 @@
...
+            <%= link_to "All Courses", user_courses_path(@user) %><br>
+            <%= link_to "Account", user_path(@user) %><br>
+            <%= link_to "Add Student", new_user_course_student_path(@user, @course) %><br>
+          </div>
+          <div class = "col-lg-4"></div>
+          <div class = "col-lg-4">
+            <h3>Flashcards</h3>
+            <div id="flashcard-info">
+              <h4 id="student-name">Name</h4>
+              <p id="student-info">info</p>
```
Comment (kcwoolverton) [inline app/views/courses/show.html.erb:39]: Capitalize I in info?

**Review by kcwoolverton — APPROVED** [review]

File: app/views/courses/show.html.erb (hunk 1 above)
Comment (Omicron3141) [reply app/views/courses/show.html.erb:6]: 4 means "AJAX response received" and 200 is the HTML status code for "no errors, returning requested information"

File: app/views/courses/show.html.erb (hunk 2 above)
Comment (Omicron3141) [reply app/views/courses/show.html.erb:39]: That part is just temporary, we will probably pre-draw a student in the future

**Pull request merged**